packs/spell-effects/spell-effect-evolution-surge-sight.json

```diff
@@ -4,7 +4,7 @@
     "name": "Spell Effect: Evolution Surge (Sight)",
     "system": {
         "description": {
-            "value": "<p>Granted by <em>@UUID[Compendium.pf2e.spells-srd.Item.Evolution Surge]</em></p>\n<p>You flood your eidolon with power, creating a temporary evolution in your eidolon's capabilities. Choose one of the following effects:</p>\n<ul>\n<li>Your eidolon gains @UUID[Compendium.pf2e.ancestryfeatures.Item.Low-Light Vision] and @UUID[Compendium.pf2e.ancestryfeatures.Item.Darkvision].</li>\n</ul>"
+            "value": "<p>Granted by <em>@UUID[Compendium.pf2e.spells-srd.Item.Evolution Surge]</em></p>\n<p>You flood your eidolon with power, creating a temporary evolution in your eidolon's capabilities. Choose one of the following effects:</p>\n<ul>\n<li>Your eidolon gains @UUID[Compendium.pf2e.ancestryfeatures.Item.Low-Light Vision] and darkvision.</li>\n</ul>"
```

Comment:
Perhaps @UUID[Compendium.pf2e.ancestryfeatures.Item.Low-Light Vision] should be stripped as well?

Reply:
and greater darkvision